site/content/zh-CN/docs/concepts/workload_priority_class.md

```diff
@@ -40,11 +40,8 @@ spec:
 ...
 ```
 
-Kueue generates the following `Workload` for the Job above.
-The `PriorityClassName` field can accept either `PriorityClass` or
-`WorkloadPriorityClass` name as a value. To distinguish, when using `WorkloadPriorityClass`,
-a `priorityClassSource` field has the `kueue.x-k8s.io/workloadpriorityclass` value.
-When using `PriorityClass`, a `priorityClassSource` field has the `scheduling.k8s.io/priorityclass` value.
+Kueue 会为上述 Job 生成如下 `Workload`。
+`PriorityClassName` 字段可以接受 `PriorityClass` 或 `WorkloadPriorityClass` 的名称。为区分两者，当使用 `WorkloadPriorityClass` 时，`priorityClassSource` 字段为 `kueue.x-k8s.io/workloadpriorityclass`；当使用 `PriorityClass` 时，`priorityClassSource` 字段为 `scheduling.k8s.io/priorityclass`。
```

Comment:
```suggestion
`PriorityClassName` 字段可以接受 `PriorityClass` 或 `WorkloadPriorityClass` 的名称。
为区分两者，当使用 `WorkloadPriorityClass` 时，`priorityClassSource` 字段为
`kueue.x-k8s.io/workloadpriorityclass`；当使用 `PriorityClass` 时，`priorityClassSource`
字段为 `scheduling.k8s.io/priorityclass`。
```
We can follow the english style to provide newline chars, to provide convenience for later maintenance.